broker-plugin/broker-cli/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>io.enmasse</groupId>
+    <artifactId>broker-plugin</artifactId>
+    <version>0.30-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>broker-cli</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.enmasse</groupId>
+      <artifactId>amqp-utils</artifactId>
```

Comment:
The JAR file for this is produced for Java 11. Is this going to be okay with Broker's JVM election?